### PR TITLE
Adds number of unread conversations to `Me`

### PR DESCRIPTION
**Pull request by zephraph**
To show a notification in the header that you have unread inquiry conversation messages we need a mechanism in the root of `Me` to be able to query the total number of conversations with unread messages. That's what this PR adds. 

I still need to follow up with a test, but I'd like feedback on the approach. 

We don't have a dedicated endpoint currently in impulse to just get the total number of unread messages. What I've done is called the `/conversations` endpoint and requested only 1 conversation (which we don't even read) as a semi-fastish way to get the unread count. Alternatively we could add a dedicated endpoint to impulse for just this thing...

Thoughts?

### CODE REVIEW

**Comment by jpotts244** [comment]
LGTM - it is nice that `total_unread_count` is at the highest level of the `/conversations` response.

File: src/schema/v2/me/index.ts [hunk 1]
```diff
@@ -138,6 +139,20 @@ const Me = new GraphQLObjectType<any, ResolverContext>({
         })
       },
     },
+    unreadConversationCount: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: "The count of conversations with unread messages.",
+      resolve: (_root, _options, { conversationsLoader }) => {
+        if (!conversationsLoader) return 0
```
Comment (mzikherman) [inline src/schema/v2/me/index.ts:146]: Nice, non-nullable!

File: src/schema/v2/me/index.ts [hunk 2]
```diff
@@ -138,6 +139,20 @@ const Me = new GraphQLObjectType<any, ResolverContext>({
...
+        const { page, size } = convertConnectionArgsToGravityArgs({
```
Comment (mzikherman) [inline src/schema/v2/me/index.ts:147]: This feels indirect (to derive `page` and `size` from a 'fake' connection-style arg of `first: 1`). Usually this helper is used for connection resolvers which do accept those connection-style arguments (typically from Relay containers).

You can prob leave out `page` (Impulse would default to 1), and `size` of 0 might work better than 1 (does Impulse work w/ that? every little bit helps!).

File: _schemaV2.graphql [hunk 3]
```diff
@@ -1010,6 +1010,9 @@ type ArtistTargetSupplyMicrofunnel {
   artworks(
     # Randomize the order of artworks for display purposes.
     randomize: Boolean
+
+    # Number of artworks to return
+    size: Int
```
Comment (mzikherman) [inline _schemaV2.graphql:1015]: possibly uniintentional?

File: src/schema/v2/me/index.ts (hunk 2 above)
Comment (zephraph) [reply src/schema/v2/me/index.ts:147]: I'll test that out!

File: _schemaV2.graphql (hunk 3 above)
Comment (zephraph) [reply _schemaV2.graphql:1015]: Will double-check

**Pull request merged**